src/brain/issueLabelHandler/route.ts

```diff
@@ -101,14 +91,11 @@ export async function markUnrouted({
     labels: [UNROUTED_LABEL, WAITING_FOR_SUPPORT_LABEL],
   });
 
-  const timeToRouteBy = await calculateSLOViolationRoute(UNROUTED_LABEL);
-  const { readableDueByDate, lastOfficeInBusinessHours } =
-    await getReadableTimeStamp(timeToRouteBy, UNKNOWN_LABEL);
   await octokit.issues.createComment({
     owner,
     repo: repo,
     issue_number: issueNumber,
-    body: `Assigning to @${SENTRY_ORG}/support for [routing](https://open.sentry.io/triage/#2-route), due by **<time datetime=${timeToRouteBy}>${readableDueByDate}</time> (${lastOfficeInBusinessHours})**. ⏲️`,
+    body: `Assigning to @${SENTRY_ORG}/support for [routing](https://open.sentry.io/triage/#2-route) ⏲️`,
```

Comment:
I thought the point of this PR was to reduce comment spam. Aren't we still commenting here, just without the due date info? Am I reading this right?

Reply:
Reminds me: how does it show up when we change a field in a project? Does that show up inline on the ticket history at all?

Reply:
I just tested out on https://github.com/getsentry/self-hosted/issues/2205#event-553709b5-b6ef-500f-808e-281bb5f8b55a. I think we get status changes in issue history but not other field changes (I set a value for "Respond By" and am not seeing it in the history).

Reply:
So we'll get a comment every time an issue changes product area but not every time an issue requires a response from that product area. Something like that?

Reply:
> So we'll get a comment every time an issue changes product area but not every time an issue requires a response from that product area. Something like that?

Yep

> I think we get status changes in issue history but not other field changes (I set a value for "Respond By" and am not seeing it in the history).

and yes

Reply:
> a comment every time an issue changes product area

I guess that works for now. We can dial back the comments further if we want in the future.